.github/workflows/build-test-virtual.yml

```diff
@@ -27,7 +27,7 @@ jobs:
       # Work-around for https://github.com/actions/runner/issues/2033
       - name: Work around git safe.directory in container
         run: chown -R $(id -u):$(id -g) $PWD
-      - run: apt-get update && apt-get install -y libcurl4-openssl-dev faketime clang-tidy-10
+      - run: apt-get update && apt-get install -y libcurl4-openssl-dev faketime clang-tidy-12
```

Comment:
why are you changing this? Is it related to the move to 3.0.13?

Reply:
not necessarily, just wanted to use a more recent version. The format errors currently flagged in the CI pipeline are occurring also with `clang-tidy-10` , I'm working on those rn